src/dev-app/drag-drop/drag-drop-demo.ts

```diff
@@ -21,6 +21,7 @@ import {CdkDragDrop, moveItemInArray, transferArrayItem} from '@angular/cdk/drag
 })
 export class DragAndDropDemo {
   axisLock: 'x' | 'y';
+  dragStartDelay: number = 0;
```

Comment:
You can omit the `: number` here. We're only explicit about it in the source code so that our doc generation tooling can pick it up.